### PR TITLE
MUMUP-2293 && MUMUP-2294 : sidebar changes

### PR DESCRIPTION
**Pull request by timlevett**
+ Remove big sidebar
+ Remove beta features that had to do with sidebar
+ Change mobile sidebar to static home/logout links
+ Removed sidebar service
+ Removed sidebar directive
+ Removed sidebar controller
+ Removed sidebar less

http://goo.gl/JDSUF9

http://goo.gl/2FIVN0

### CODE REVIEW

File: uw-frame-components/js/frame-config.js [hunk 1]
```diff
@@ -33,9 +33,9 @@ define(['angular'], function(angular) {
         })
         .constant('FRAME_BETA_FEATURES', [
           {
-            "id" : "sidebarShowSettings",
-            "title" : "Sidebar Settings",
-            "description" : "Shows the settings button to get to this page easier"
+            "id" : "showSettings",
+            "title" : "Beta Settings",
+            "description" : "Shows/Hides the Beta Settings"
```
Comment (vertein) [inline uw-frame-components/js/frame-config.js]: Is this used anywhere?

Comment (timlevett) [inline uw-frame-components/js/frame-config.js]: Not currently, was thinking of putting it in the options menu

Comment (vertein) [inline uw-frame-components/js/frame-config.js]: Works for me.

**Comment by vertein** [comment]
:+1:

**Comment by jhanstra** [comment]
:+1:

**Pull request merged**